packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/DocumentAsset.java

```diff
@@ -3,13 +3,17 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.annotations.TSOptional;
 
 import java.io.Serializable;
 
 @Data
 @Accessors(chain = true)
-public class Publication implements Serializable {
+@TSModel
+public class DocumentAsset implements Serializable {
```

Comment:
`implements Serializable` we don't need them.

Reply:
Will do this as seperate issue